src/main/keystore/generate_self_signed_cert.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+cd "$(dirname "$0")/../.."
+
+tmpDir=$(mktemp -d)
+trap 'rm -rf $tmpDir' EXIT
+
+cat > "$tmpDir/req.cnf" <<EOF
+[ req ]
+default_bits 		= 2048
+prompt 				= no
+distinguished_name 	= req_distinguished_name
+x509_extensions 	= req_ext
+
+[ req_distinguished_name ]
+countryName         	= US
+stateOrProvinceName 	= NY
+localityName 			= New York
+organizationName    	= Jenkins
+organizationalUnitName 	= Test
+commonName          	= Self-Signed CA
+emailAddress			= noreply@jenkins.io
+
+[ req_ext ]
+keyUsage				= digitalSignature
+basicConstraints		= CA:false
+subjectAltName 			= @alternate_names
+subjectKeyIdentifier 	= hash
+
+[ alternate_names ]
+DNS.1         = localhost
```

Comment:
I thought we would include at least `*.localtest.me`?

Reply:
Not needed for now.